### DISCUSSION TITLE
Is it possible to get blockable in the block editor?

### DISCUSSION THREAD

**Discussion by jamestylerpatton** [area17/twill#865] [Q&A]
`$block->blockable` doesn't seem to be available to me when previewing the block in the block editor.

Let's say I have a Page form with a title input and a block editor, and within one of the blocks I need to display the page title, is it possible to get a reference to that Page to get the title?

Thanks!

**Comment by ptrckvzn** [comment]
Hello @jamestylerpatton, I think you are right. In page preview, the `blockable` relationship seems available, but not in the block editor. I will tag this as something we can work out eventually - hopefully soon.

**Comment by jamestylerpatton** [comment]
@ptrckvzn I was able to work around this by passing the blockable id as a hidden input in the block form and creating a view composer to query the blockable object if `$block->blockable` is not available in the view.

It's not ideal and causes issues when trying to duplicate modules that use these blocks, but it's better than nothing.

**Comment by michaeljcoyne** [comment]
@jamestylerpatton how do did you do the markup in the form?
